downloads/metadata/public-financing-fields.csv

```diff
@@ -0,0 +1,4 @@
+---
+---
+Field Name,Question,Short Label,Additional Notes{% for control in site.data.sections.["public-financing"].controls %}
```

Comment:
Are there any weird characters in _data/sections/public-financing/controls.yml by any chance?